app/lib/esClient.js

```diff
@@ -0,0 +1,33 @@
+const elasticsearch = require('elasticsearch');
+const esConfig = require('../../config/config').es;
+
+const client = elasticsearch.Client({ host: `${esConfig.host}:${esConfig.port}` });
+function getByIdQuery(id) {
+  return {
+    index: 'profiles',
```

Comment:
@markysoft Is it possible to define the ES index as an environment variable also? Don't mind the default been profiles though.